Beta-Game/Beta-Game/main.swift

```diff
@@ -0,0 +1,708 @@
+printPrefQuestion()
+
+//  store user reponse in var to use for comparisons
+var pref = prefAnswer()
+
+/*
+ *  repeat to enter for options first time around and loop if
+ *  user enters invalid response
+ */
+repeat  {
+    /*
+     *  Unwrap due to readline know it's going to string to compare and
+     *  lowercase
+     */
+    if pref?.lowercased() == "a" {  //**Maybe Switch Instead**
+        printMaleChars()
+        // sets badResonse to false to exit
+        badResponse = false
+    }
+
+    else if pref?.lowercased() == "b"    {
+        printFemaleChars()
+        badResponse = false
+    }
+
+    //  Don't get past this till valid response
+    else    {
+        print("\nCome Again?")
+        printPrefQuestion()
+        pref = prefAnswer()
+            /*  if user still enters invalid response badRespnse stays true
+            *  will run loop to recheck for response
+            */
+            if pref?.lowercased() != "a" || pref?.lowercased() != "b"   {
+                badResponse = true
+            }
+    }
+} while badResponse == true
+
+//-------------------------------------------------
+
+//var charSel = prefAnswer()
+
+//var choice = (pref?.lowercased(), charSel?.lowercased())
+
+//let romeo = ("a", "a") ?? : String
+//let juliet = ("b", "a")
+
+repeat  {
+    let charSel = prefAnswer()
+    let choice = (pref?.lowercased(), charSel?.lowercased())
+    switch choice   {
+    case ("a", "a") :
+        print("\nYou Chose Romeo!")
+        playerCharChoice = romeo
+        badResponse = false
+    case ("b", "a") :
+        print("\nYou Chose Juliet!")
+        playerCharChoice = juliet
+        badResponse = false
+    default:
+        print("\n(Sorry what?)")
+        if pref?.lowercased() == "a"  {
+            printMaleChars()
+            //charSel = prefAnswer()
+            //print(charSel.lowercased())
+            if charSel?.lowercased() != "a"   {//print("here")
+                    badResponse = true
+                }
+                //else {choice = ("a", "a")}
+        }
+    
+        else if pref?.lowercased() == "b"   {
+            printFemaleChars()
+            //charSel = prefAnswer()
+            if charSel?.lowercased() != "a"   {
+                badResponse = true
+            }
+            //else {choice = ("b", "a")}
+        }
+    }
+} while badResponse == true
+var pickUpLine = ""
+func encounter(playerChoice : String) -> String   {
+    //var playerChoice = playerCharChoice
+    print("\n*************Encounter Phase**************")
+    print("\nSaw \(playerChoice), enter pick up line:")
+    pickUpLine = readLine() ?? ""
+    let lineLength = pickUpLine.count
+    
+    for _ in 1...lineLength {print("🔥", terminator:"")}
+    
+    print("\n\n\(pickUpLine )? Interesting approach. What's your name?\n")
+    let playerName = readLine()
+    print("\nWell My name is \(playerChoice).")
+    print("\nAnd how old are you?")
+    guard let age = readLine() else { return "" }
```

Comment:
Good use of a guard statement, however this doesn't account for when a person inputs anything other than an Int, it defaults to the ) "0 < 26" bracket.